Form/Type/VichFileType.php

```diff
@@ -9,31 +9,37 @@
 use Symfony\Component\Form\FormEvents;
 use Symfony\Component\Form\FormInterface;
 use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\Options;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+use Symfony\Component\PropertyAccess\PropertyPath;
 use Vich\UploaderBundle\Form\DataTransformer\FileTransformer;
 use Vich\UploaderBundle\Handler\UploadHandler;
+use Vich\UploaderBundle\Mapping\PropertyMappingFactory;
 use Vich\UploaderBundle\Storage\StorageInterface;
 
+/**
+ * @author Kévin Gomez <contact@kevingomez.fr>
+ * @author Konstantin Myakshin <koc-dp@yandex.ru>
+ * @author Massimiliano Arione <max.arione@gmail.com>
+ */
 class VichFileType extends AbstractType
 {
-    /**
```

Comment:
I'm not sure that every IDE is smart enough to pass the type after assignment. Anyway, phpdoc block could be used for other than autocompletion in IDEs (e.g. for doc auto-generation).
Anyway, it would be better to keep consistency between files (so, since other files do use phpdoc blocks, this file should use it as well)